specification/cognitiveservices/data-plane/Speech/SpeakerRecognition/identification/identification-swagger.json

```
@@ -0,0 +1,771 @@
+            }
+          },
+          "401": {
+            "$ref": "#/responses/Unauthorized"
+          },
+          "404": {
+            "$ref": "#/responses/NotFound"
+          },
+          "429": {
+            "$ref": "#/responses/RateLimit"
+          },
+          "500": {
+            "$ref": "#/responses/ServerError"
+          }
+        }
+      },
+      "delete": {
+        "description": "Deletes an existing profile.",
+        "summary": "Delete Profile",
+        "tags": [
+          "text-independent"
+        ],
+        "responses": {
+          "204": {
+            "description": "OK"
+          },
+          "401": {
+            "$ref": "#/responses/Unauthorized"
+          },
+          "429": {
+            "$ref": "#/responses/RateLimit"
+          },
+          "500": {
+            "$ref": "#/responses/ServerError"
+          }
+        }
+      }
+    },
+    "/text-independent/profiles/{profileId}/reset": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/profileIdParam"
+        }
+      ],
+      "post": {
+        "description": "Resets existing profile to its original creation state. The reset operation does the following:\n* Updates enrollmentStatus to Enrolling.\n* Updates lastUpdatedDateTime.\n* Updates enrollmentsCount to 0.\n* Updates enrollmentsLength to 0.\n* Updates enrollmentsSpeechLength to 0.\n* Updates remainingEnrollmentsSpeechLength to the required number.\n* Removes all associated enrollments from storage.\n* Removes chosen passphrase association.\n* Resets value of modelVersion.",
+        "summary": "Reset Profile",
+        "tags": [
+          "text-independent"
+        ],
+        "responses": {
+          "204": {
+            "description": "OK"
+          },
+          "401": {
+            "$ref": "#/responses/Unauthorized"
+          },
+          "404": {
+            "$ref": "#/responses/NotFound"
+          },
+          "429": {
+            "$ref": "#/responses/RateLimit"
+          },
+          "500": {
+            "$ref": "#/responses/ServerError"
+          }
+        }
+      }
+    },
+    "/text-independent/profiles/{profileId}/enrollments": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/profileIdParam"
+        }
+      ],
+      "post": {
+        "description": "Adds an enrollment to existing profile.\nThe first enrollment must be a predefined activation phrase which can be lised using the /phrases/{locale} api.\nIf the minimum number of requested enrollment audios is reached, a voice print is created.\nAny further enrollemnt will be used to improve the voice print.\n\nLimitations:\n* Minimum amount of speech needed for voice print creation is **20 seconds**\n* Maximum enrollment audio length is **120 seconds**\n* Minimum enrollment audio length is **1 second**\n* Maximum length allowed across all enrollments is **300 seconds**\n* Minimum audio Signal-to-noise ratio (SNR) is **2dB**",
```

```suggestion
        "description": "Adds an enrollment to existing profile.\nThe first enrollment must be a predefined activation phrase which can be listed using the /phrases/{locale} api.\nIf the minimum number of requested enrollment audios is reached, a voice print is created.\nAny further enrollement will be used to improve the voice print.\n\nLimitations:\n* Minimum amount of speech needed for voice print creation is **20 seconds**\n* Maximum enrollment audio length is **120 seconds**\n* Minimum enrollment audio length is **1 second**\n* Maximum length allowed across all enrollments is **300 seconds**\n* Minimum audio Signal-to-noise ratio (SNR) is **2dB**",
```

corrected spelling of "listed" and "enrollement"

---

done

---

@yshahin isn't it "enrollment" instead of "enrollement"?

---

@mikekistler @

> @markweitzel @mikekistler @anuchandy can I use openapi v3 swagger or should I keep it v2?

Also does this repo support yaml format?

---

> @yshahin isn't it "enrollment" instead of "enrollement"?

I already fixed it with the correct spelling. If it was incorrect then the cspell checker build pipeline would have failed.

---

> > @yshahin isn't it "enrollment" instead of "enrollement"?
> 
> I already fixed it with the correct spelling. If it was incorrect then the cspell checker build pipeline would have failed.

Thanks Youssef.

---

> @mikekistler @
> 
> > @markweitzel @mikekistler @anuchandy can I use openapi v3 swagger or should I keep it v2?
> 
> Also does this repo support yaml format?

I used to upload yaml format, API Reference docs guy told me that they only accept json format.

---

I think our current tooling only supports OpenAPI v2 in json format.